pallets/native-token-management/src/lib.rs

```diff
@@ -138,6 +141,13 @@ pub mod pallet {
 			token_amount: NativeTokenAmount,
 		) -> DispatchResult {
 			ensure_none(origin)?;
+			Initialized::<T>::mutate(|initialized| {
+				if !*initialized {
+					log::info!("");
```

Comment:
What is the purpose of the empty log? Is it so the code contains I/O and doesn't get optimized away or something?

Reply:
Nah, just something I forgot to remove